robo.yml

```diff
@@ -88,3 +88,8 @@ data:
     graph: "http://publications.europa.eu/resource/dataset/role-qualifier"
     url: "http://publications.europa.eu/resource/cellar/b897302e-ce0b-11eb-ac72-01aa75ed71a1.0001.03/DOC_1"
     format: "rdf"
+
+  - name: "strategic-priority"
+    graph: "http://publications.europa.eu/resource/authority/strategic-priority"
+    url: "https://publications.europa.eu/resource/cellar/ea0b97ae-20f4-11ec-bd8e-01aa75ed71a1.0001.03/DOC_1"
```

Comment:
Usually we use more full version: strategic-priority-skos-ap-act.rdf
```suggestion
    url: "http://publications.europa.eu/resource/cellar/ea0b97ae-20f4-11ec-bd8e-01aa75ed71a1.0001.01/DOC_1"
```
Also during testing, I've noticed that we again have a senseless term:
![image](https://user-images.githubusercontent.com/9432036/150797398-6b4eea0b-5f10-4280-8471-1adead505d4d.png)